plat/nxp/common/layerscape/README.TRUSTED_BOOT

```diff
@@ -0,0 +1,81 @@
+TRUSTED_BOARD_BOOT option can be enabled by specifying
```

Comment:
This file should go in the ``docs/plat`` folder. Maybe you can create a subfolder called ``layerscape`` if you are going to have more than one file.

Reply:
Moved to docs/plat/nxp/nxp-layerscape-tb.rst